### DISCUSSION TITLE
Download/Deploy local process group anomaly settings

### DISCUSSION THREAD

**Discussion by maximoscardi** [Dynatrace/dynatrace-configuration-as-code#443] [Ideas]
The method _/anomalyDetection/processGroups/{id}_ gets the configuration of anomaly detection for the specified process group

Would it be feasible to include in Monaco the possibility to export all specific local process group settings ?

**Comment by mdicss** [comment]
It would be great to be able to use all the anomaly settings in monaco e.g. global anomaly detection for services and overrides for defined services. All the other stuff in the /anomalyDetection/...-api too.
